src/stcal/jump/twopoint_difference.py

```diff
@@ -255,6 +281,52 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
                             if (gdq[integ, group, row, col + 1] & dnu_flag) == 0:
                                 gdq[integ, group, row, col + 1] =\
                                     np.bitwise_or(gdq[integ, group, row, col + 1], jump_flag)
+
+        # flag n groups after all jumps to account for the transient seen
+        flag_dn_threshold = [after_jump_flag_e1, after_jump_flag_e2]
+        flag_groups = [after_jump_flag_n1, after_jump_flag_n2]
+        # ensure the smallest threshold is 1st
```

Comment:
why is this commented out? maybe we should give a warning if someone tries to pass in thresholds that are out of order?

Reply:
Was commented out as it was part of an attempt to make more efficient code that failed.  For the current code, it does not matter what order the thresholds are in.